extensions/amp-recaptcha-input/0.1/test/test-amp-recaptcha-input.js

```diff
@@ -32,19 +32,35 @@ describes.realWin('amp-recaptcha-input', {
     toggleExperiment(win, 'amp-recaptcha-input', true);
   });
 
-  function getRecaptchaInput(skipSiteKey, skipAction) {
+  function getRecaptchaInput(config) {
     const ampRecaptchaInput = doc.createElement('amp-recaptcha-input');
     ampRecaptchaInput.setAttribute('layout',
         'nodisplay');
-    if (!skipSiteKey) {
+
+    if (config) {
+      if (!config.skipName) {
```

Comment:
nit: [Flat is better than nested](https://en.wikibooks.org/wiki/Computer_Programming/Coding_Style/Minimize_nesting). Avoid nesting if blocks, and nesting blocks generally. It tends to make code harder to understand.



```js
function getRecaptchaInput(config = {}) {
  /* ... */

  if (!config.skipName) {
    ampRecaptchaInput.setAttribute('name', 'recaptcha-test');
  }

  /* ... */
}
```

Reply:
Didn't know our compiler would handle the default param assignment thingy (don't know the technical term off of the top of my head haha!), will definitely use that, thanks! 😄

Reply:
np! you can do the same thing with `config = config || {};`